flink-scala-shell/start-script/start-scala-shell.sh

```diff
@@ -19,6 +19,9 @@
 
 # from scala-lang 2.10.4
 
+# Uncomment the following line to enable remote debug
+# export FLINK_SCALA_SHELL_JAVA_OPTS="-agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=5005"
```

Comment:
If not remote debug mode,  it does not exist the env variable `FLINK_SCALA_SHELL_JAVA_OPTS `, but it would be used in other place. What's more, I think use a bool flag would be better that "uncomment".

Reply:
I don't find something like `flink-env.sh` for specify flink related env variable. So I just put it in `start-scala-shell.sh` directly. I don't think bool flag is proper, as it hide the details of remote debug info, like which port to listen. I believe most of apache project use `*JAVA_OPTS` for remote debug which is very common. What do you think ?

Reply:
I know there is not a env variable named "FLINK_SCALA_SHELL_JAVA_OPTS" before you create. I said, it is commented, but you always used in "java" command later.
About the bool flag, I mean we could add a flag named "remoteDebugMode" to control the "export" operation. 

for example : 

```
if $remoteDebugMode then 
export FLINK_SCALA_SHELL_JAVA_OPTS="-agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=5005"
fi
```

It's better than "comment" & "uncomment". Why do you think it hides the details of remote debug info?

Reply:
Sorry @yanghua I still think using bool flag is not proper. Regarding `hiding the details of remote debug info`, I mean bool flag only control whether to enable remote debug, but does't control how, such as which port to listen.  If we introduce this bool flag,  we will use 2 env variable to control whether and how to remote debug which is too verbose to me.

Reply:
OK, I just think using a comment which may been ignored by the user. And using a bool flag which also can control whether add  `FLINK_SCALA_SHELL_JAVA_OPTS ` as a option to java command or not. Please feel free to follow your idea.